benchmark/url/url-searchparams-iteration.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const common = require('../common.js');
+const URLSearchParams = require('url').URLSearchParams;
+const v8 = require('v8');
+
+v8.setFlagsFromString('--allow_natives_syntax');
+
+const bench = common.createBenchmark(main, {
+  method: 'forEach iterator'.split(' '),
```

Comment:
This was from the existing [url/new-url-parse.js](https://github.com/nodejs/node/blob/4b312387ead4ba11146b28b8ac05ed385919c4af/benchmark/url/new-url-parse.js#L7) benchmark.